### PR TITLE
Two moon board changes

### PR DESCRIPTION
**Pull request by kberg**
1. The second colony rate reward is now 1 energy production. This required a lot of css changes to shift everything to the right.
2. The sidebar values are in the same order with the  three graphs, plus a little color.

### CODE REVIEW

**Comment by kberg** [comment]
image

**Comment by kberg** [comment]
image

**Comment by kberg** [comment]
Fixes #3398

File: src/components/MoonGlobalParameterValue.ts [hunk 1]
```diff
@@ -29,7 +25,9 @@ export const MoonGlobalParameterValue = Vue.component('moon-global-parameter-val
       <img src="/assets/misc/checkmark.png" class="checkmark" :alt="$t('Completed!')">
     </div>
     <div v-else class="moon_params_value">
-      {{this.value()}}
+      <span class="colony">{{this.moonData.colonyRate}}</span>
```
Comment (bafolts) [inline src/components/MoonGlobalParameterValue.ts:28]: We do it all the time but fwiw I don't think it is recommended to use `this` within a vue template. The linter will eventually enforce not using `this.` in templates once we update to the recommended vue rules.

**Review by bafolts — APPROVED** [review]

File: src/components/MoonGlobalParameterValue.ts (hunk 1 above)
Comment (kberg) [reply src/components/MoonGlobalParameterValue.ts]: OK I removed it and playtested it4.

**Pull request merged**